src/nvim/main.c

```diff
@@ -1217,6 +1217,11 @@ static void command_line_scan(mparm_T *parmp)
     }
   }
 
+  if (embedded_mode && exmode_active && silent_mode) {
+    mch_errmsg(_("Error: flags -es and -Es are incompatible with --embed.\n"));
+    mch_exit(2);
```

Comment:
"Error:" prefix is not consistent with existing startup failure messages, cf. https://github.com/neovim/neovim/blob/3afb397407af/src/nvim/main.c#L136-L142

So I removed it. Also, other messages use the term "arguments", not "flags".

Also `mainerr()` should be used.